Runtime/Game/Requests/LeaderboardRequest.cs

```diff
@@ -73,6 +73,105 @@ public class LootLockerSubmitScoreResponse : LootLockerResponse
         public string metadata { get; set; }
     }
 
+    public class LootLockerLeaderboardHistoryResponse : LootLockerResponse
```

Comment:
I realized a small thing. It's called Archive consistently except for here, where it's called History. We should settle for 1 term.